src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -406,6 +412,69 @@ def _yangConvert(val):
 
         return vValue
 
+    """
+    Xlate a Qos Maps list
+    This function will xlate from a dict in config DB to a Yang JSON list
+    using yang model. Output will be go in self.xlateJson
+
+    Note: Exceptions from this function are collected in exceptionList and
+    are displayed only when an entry is not xlated properly from ConfigDB
+    to sonic_yang.json.
+
+    QOS MAPS Yang has netsted list, which is diffrent from config DB.
+    Each field value in config db should be converted to netsted list with
+    key and value.
+    """
```

Comment:
Plz put  example of actual config....both configDB and yang.  Better while referring code later by any Dev. Again thx for addressing it.

Reply:
Done

Reply:
Sorry, I meant yang.json example. Thx

Reply:
Done